### ISSUE TITLE
outcome problem in test.py with your model.

### ISSUE THREAD

**Issue by dobbleheart** [KangLiao929/DR-GAN#6]
![0](https://user-images.githubusercontent.com/45570026/203218350-4f3f6082-d56f-40e5-a761-e4efe8ed0a2f.jpg)

This  is the test.py outcome in DR-GAN/datasets/pre. Please tell me the mistake i have made during run your code, thank you very much! I think it is my mistake , but i cant find the way to deal with it. Please help me ,thank you.

**Comment by KangLiao929** [comment]
Hi, did you train DR-GAN by yourself? It looks a small bug in inference process such as the image normalization.

**Comment by dobbleheart** [comment]
Hello , thank you for giving me back.


&nbsp; &nbsp; &nbsp; i use the pre-trained model in the code repository with generator.h5.&nbsp; &nbsp;I just test the picture in the DR-GAN-main/dataset/B/0.jpg.


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "KangLiao929/DR-GAN"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年11月25日(星期五) 上午10:24
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [KangLiao929/DR-GAN] outcome problem in test.py with your model. (Issue #6)





 
Hi, did you train DR-GAN by yourself? It looks a small bug in inference process such as the image normalization.

**Comment by KangLiao929** [comment]
You are welcome. I tested the pre-trained model and it still works for me. I would like to suggest you to try the train.py and see how's the intermediate training results going. The training for DR-GAN is very fast.

**Comment by dobbleheart** [comment]
Thank you very much for your advice, I will try to study your code, it may be a bit difficult for me. I hope I can continue to ask you for advice when I have questions afterwards, and I wish you well in your research and good health.


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "KangLiao929/DR-GAN"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年11月25日(星期五) 中午11:13
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [KangLiao929/DR-GAN] outcome problem in test.py with your model. (Issue #6)





 
You are welcome. I tested the pre-trained model and it still works for me. I would like to suggest you to try the train.py and see how's the intermediate training results going. The training for DR-GAN is very fast.

**Comment by KangLiao929** [comment]
No problem, just leave a comment here if you got some issues. Thanks for your words and I wish you everything goes well.

**Comment by lwttttt** [comment]
I did not train it by myself, but still got the same problem.........is it because the pretrained model you offered before?

**Comment by lwttttt** [comment]
And the new link of the pretrained model you offered can not be tested, it showed that:
<type float 32> has shape(4,4,64,128), but the saved weight has shape (64,3,4,4)

**Comment by KangLiao929** [comment]
> And the new link of the pretrained model you offered can not be tested, it showed that: <type float 32> has shape(4,4,64,128), but the saved weight has shape (64,3,4,4)

This issue could be referred to https://github.com/qqwweee/keras-yolo3/issues/417.
